cardano-cli/test/cardano-cli-golden/files/golden/help/allegra_query_stake-address-info.cli

```diff
@@ -1,11 +1,11 @@
 Usage: cardano-cli allegra query stake-address-info --socket-path SOCKET_PATH
                                                       [--cardano-mode
                                                         [--epoch-slots SLOTS]]
-                                                      --address ADDRESS
                                                       ( --mainnet
                                                       | --testnet-magic NATURAL
                                                       )
                                                       [--out-file FILEPATH]
+                                                      --address ADDRESS
```

Comment:
This is the downside of the refactor. It affects the parsers and I would argue having `--out-file` as the last argument is more natural. More and more I wish for monadic parsers.

Reply:
Yes I agree too and was wondering after the initial golden files changes myself.

I pulled `mOutFile` out of the `QueryCommons` type and made sure that `--out-file` is the last argument in every query :+1: